docs/manual_upgrades.rst

```diff
@@ -8,6 +8,31 @@ Please add new instructions to the top, include a date, and make a post in the `
 (If you just need to update your devstack to the latest version of everything, see :doc:`updating_devstack`.)
 
 
+2024-04-29 - Moved Open edX repositories
+****************************************
+
+Several repositories were being deprecated by Open edX (`issue #227 <https://github.com/openedx/public-engineering/issues/227>`_).
+We are forking some of those repositories to keep maintaining them.
+
+If you use any of the following repositories, please make sure to update the remote URLs:
+
+- **edx-analytics-dashboard**::
+
+    git remote set-url origin git@github.com:edx/edx-analytics-dashboard.git
```

Comment:
Not sure how you want to represent the devstack working dir in the following, but do you want some version of the following before each? Not sure if you even need the headers, or if you just want one large code block. Your decision.
```
cd edx-analytics-dashboard
```

Reply:
Updated instructions, maybe it's clearer now. What do you think?